src/app/build/build.component.html

```diff
@@ -0,0 +1,197 @@
+<div class="grid-container build-chart-" ng-class="{'active-chart': showChart }">
+
+  <div class="grid-content build-header-wrap">
+    <div class="grid-block horizontal">
+      <div class="small-12">
+
+        <div class="build-header">
+          <h1>
+            <a href="../kashti/#!/project/brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac" title="Back to Build list"><i class="icon ios-arrow-back-outline"></i></a>
+            Build &nbsp;<em class="ng-binding">01bzzn45f0p5bjtsffw9m8chts</em>
+          </h1>
+
+          <div class="build-status success" ng-class="{
+            'success': build.worker.status == 'Succeeded',
+            'failed': build.worker.status == 'Failed',
+            'active': build.worker.status == 'Pending'}" style="">
+            <div class="success" ng-show="build.worker.status == 'Succeeded'" style=""><i class="icon ion-checkmark-circled"></i> Passed</div>
+            <div class="failed ng-hide" ng-show="build.worker.status == 'Failed'"><i class="icon md-close-circle"></i> Failed</div>
+            <div class="active ng-hide" ng-show="build.worker.status == 'Pending'"><img src="assets/images/rolling.svg"> In Progress</div>
+          </div>
```

Comment:
Build status might be useful as a standalone component that we could include elsewhere, like the dashboard and here. Just thinking out loud, not needed for this PR.

Reply:
+1